include/CppUTest/CppUTestConfig.h

```diff
@@ -216,8 +218,6 @@ typedef struct { char dummy[16]; } cpputest_ulonglong;
 typedef struct { char dummy[8]; } cpputest_longlong;
 typedef struct { char dummy[8]; } cpputest_ulonglong;
 #endif
-#define CPPUTEST_LONGLONG_DEFAULT cpputest_longlong()
-#define CPPUTEST_ULONGLONG_DEFAULT cpputest_ulonglong()
```

Comment:
Lets see if we can think of a better way than using this DEFAULT #define :) So far, it wasn't a problem (yet). The coverage decreased a bit, need to check still why. Thanks!!